language/stdlib/modules/validator_set.mvir

```diff
@@ -27,7 +33,10 @@ module ValidatorSet {
         // Only callable by the validator set address
         assert(get_txn_sender() == 0x1D8, 1);
 
-        move_to_sender<T>(T { validators: Vector.empty<Self.ValidatorInfo>() });
+        move_to_sender<T>(T {
+            validators: Vector.empty<Self.ValidatorInfo>(),
+            change_events: Event.new_event_handle<Self.ChangeEvent>(),
```

Comment:
We have to guarantee that this event is the first outbound event for validator set address(to make sure that the salt is indeed 0). Do we need some test on that to make sure we don't mess that up?

Reply:
Yep, this diff includes a test for that (see assertions in genesis creation code).